### DISCUSSION TITLE
Strings finalized

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enrouteText#24] [Team Posts]
Dear all,

as you probably saw, I went through the whole app in detail and changes many strings -- my goal was to use uniform language, and to shorten the strings so that the text fits even on devices with narrow screens. As a result, many strings have changed. I will now stop and turn to other parts of our project.

Best,

Stefan.

@woodwind-enr I have already updated all the German translations. I will now give the translations back to you: pleas check from time to time and update/translate as necessary.

@acrovato There are many things that need updating now. Given how bad my French is, I fear that I cannot help you much. Please have a look .

**Comment by woodwind-enr** [comment]
Wilco
vg
m


Am 7. Juli 2020 08:07:10 MESZ schrieb Stefan Kebekus <notifications@github.com>:
>Dear all,
>
>as you probably saw, I went through the whole app in detail and changes
>many strings -- my goal was to use uniform language, and to shorten the
>strings so that the text fits even on devices with narrow screens. As a
>result, many strings have changed. I will now stop and turn to other
>parts of our project.
>
>Best,
>
>Stefan.
>
>@woodwind-enr I have already updated all the German translations. I
>will now give the translations back to you: pleas check from time to
>time and update/translate as necessary.
>
>@acrovato There are many things that need updating now. Given how bad
>my French is, I fear that I cannot help you much. Please have a look .
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/orgs/Akaflieg-Freiburg/teams/enroute-translation/discussions/2

--
Vom Smartphone aus gesendet. Absonderlichkeiten bitte entschuldigen.

**Comment by acrovato** [comment]
Hi,
French translations are now updated and merged into master!  :-)
